openssl-sys/build/find_normal.rs

```diff
@@ -50,6 +50,10 @@ fn find_openssl_dir(target: &str) -> OsString {
                 return homebrew.to_path_buf().into();
             }
         }
+        let port_out = get_macport_openssl();
+        if !port_out.is_empty() {
```

Comment:
It's generally preferable to use an Option to track optional data than the emptyness of a string, IMO.